packages/react-hooks/src/useOwnStashes.ts

```diff
@@ -31,18 +31,18 @@ function getStashes (allAccounts: string[], ownBonded: Option<AccountId>[], ownL
 }
 
 export function useOwnStashes (): [string, IsInKeyring][] | undefined {
-  const { allAccounts, hasAccounts } = useAccounts();
+  const { allAccounts, areAccountsLoaded, hasAccounts } = useAccounts();
   const { api } = useApi();
   const ownBonded = useCall<Option<AccountId>[]>(hasAccounts && api.query.staking?.bonded.multi, [allAccounts]);
   const ownLedger = useCall<Option<StakingLedger>[]>(hasAccounts && api.query.staking?.ledger.multi, [allAccounts]);
 
   return useMemo(
-    () => hasAccounts
-      ? ownBonded && ownLedger
+    () => areAccountsLoaded && ownBonded !== undefined && ownLedger !== undefined
```

Comment:
Not sure what this specifically addresses? What behavior is different here? ... I'm probably slow today, so has not dawned on me by just looking :)

Reply:
Previously when accounts were not loaded it saw this as `[]` instead of undefined.
But yeah - I collapsed `ownBonded !== undefined && ownLedger !== undefined` to the prev version